### PR TITLE
musikcube: 0.96.10 -> 0.97.0

### PR DESCRIPTION
**Pull request by afh**
###### Description of changes

Update [musikcube](https://github.com/clangen/musikcube) to [0.97.0](https://github.com/clangen/musikcube/releases/tag/0.97.0) ([CHANGELOG](https://github.com/clangen/musikcube/blob/master/CHANGELOG.txt#L17-L24))

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] **No**. For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] **No packages depend on musikcube.** Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/musikcube/default.nix [hunk 1]
```diff
@@ -6,6 +6,7 @@
 , fetchFromGitHub
 , fetchpatch
 , ffmpeg
+, gnutls
```
Comment (afh) [inline pkgs/applications/audio/musikcube/default.nix:9]: ℹ️ Compilation failed with the following error without `gnutls` being in the `buildInputs`:

```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
LIBGNUTLS
    linked by target "server" in directory /tmp/nix-build-musikcube-0.97.0.drv-0/source/src/plugins/server
```

File: nixos/doc/manual/from_md/release-notes/rl-2205.section.xml [hunk 2]
```diff
@@ -1922,6 +1922,12 @@
           generating host-global NNCP configuration.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          The <link xlink:href="https://musikcube.com">musikcube</link>
```
Comment (aanderse) [inline nixos/doc/manual/from_md/release-notes/rl-2205.section.xml:1927]: We don't need to list every minor package update between versions in release notes. Please drop this.

**Review by aanderse — CHANGES_REQUESTED** [review]
Just one tiny change and then we're good to merge. Thanks!

File: nixos/doc/manual/from_md/release-notes/rl-2205.section.xml (hunk 2 above)
Comment (afh) [reply nixos/doc/manual/from_md/release-notes/rl-2205.section.xml]: Got it, thanks @aanderse.

**Comment by aanderse** [comment]
Sorry to pester but can you please do rebase instead of merge? We don't want merge commits like this in the repository. Thank you!

**Comment by afh** [comment]
Happy to, @aanderse, and apologies if I missed the note about nixpkgs' PR (squash) merge/rebase preference.

**Comment by aanderse** [comment]
Thanks!

**Comment by afh** [comment]
Thanks for merging, @aanderse.

**Pull request merged**